lib/samson/console_extensions.rb

```diff
@@ -0,0 +1,19 @@
+module Samson
+  module ConsoleExtensions
+    def login(role = Role::SUPER_ADMIN)
+      user = User.where(role_id: role.id).last!
```

Comment:
Just seems weird to specify a Role, and wind up as whoever the last person with that role is.

I'd be more inclined to do something like `login_as "jmoter@zendesk.com"`, or `login_as User.find(1234)`, to set current user as the user of your choice.